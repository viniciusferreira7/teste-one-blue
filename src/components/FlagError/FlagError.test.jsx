import { render } from '@testing-library/react';
import { FlagError } from '.';

describe('<FlagError/>', () => {
  it('should render with default values', () => {
    render(<FlagError>Texto</FlagError>);
    // expect(screen.getByRole('heading')).toBeInTheDocument();
  });
});
