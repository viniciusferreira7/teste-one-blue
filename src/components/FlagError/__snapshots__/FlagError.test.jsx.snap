// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FlagError/> should match snapshot 1`] = `
<p
  class="sc-bczRLJ bcsOYy"
>
  Erro
</p>
`;
